src/openstack_mcp_server/tools/neutron_tools.py

```diff
@@ -0,0 +1,297 @@
+from .base import get_openstack_conn
+from fastmcp import FastMCP
+from pydantic import BaseModel
+
+
+class Network(BaseModel):
+    id: str
+    name: str
+    status: str
+    description: str | None = None
+    is_admin_state_up: bool = True
+    is_shared: bool = False
+    mtu: int | None = None
+    provider_network_type: str | None = None
+    provider_physical_network: str | None = None
+    provider_segmentation_id: int | None = None
+    project_id: str | None = None
+
+
+class Subnet(BaseModel):
+    id: str
+    name: str
+    status: str
+
+
+class Port(BaseModel):
+    id: str
+    name: str
+    status: str
+
+
+class Router(BaseModel):
+    id: str
+    name: str
+    status: str
+
+
+class SecurityGroup(BaseModel):
+    id: str
+    name: str
+    status: str
+
+
+class SecurityGroupRule(BaseModel):
+    id: str
+    name: str
+    status: str
+
+
+class FloatingIP(BaseModel):
+    id: str
+    name: str
+    status: str
+
+
+class NeutronTools:
+    """
+    A class to encapsulate Neutron-related tools and utilities.
+    """
+
+    def _convert_to_network_model(self, openstack_network) -> Network:
+        """
+        Convert an OpenStack network object to a Network pydantic model.
+
+        Args:
+            openstack_network: OpenStack network object
+
+        Returns:
+            Network pydantic model
+        """
+        return Network(
+            id=openstack_network.id,
+            name=openstack_network.name or "",
+            status=openstack_network.status or "",
+            description=getattr(openstack_network, "description", None),
```

Comment:
description에 아래처럼 접근이 안되나요?

```suggestion
            description=openstack_network.description,
```

Reply:
값이 없을 경우 예외가 발생할줄 알았는데 의외로 None을 반환하네요. 수정했습니다.